test/src/main/java/com/elasticpath/cucumber/definitions/demo/HomePageDefinition.java

```diff
@@ -23,15 +23,15 @@
 
 import cucumber.api.java.Before;
 
-import com.elasticpath.selenium.framework.util.SeleniumDriverSetup;
+import com.elasticpath.selenium.SetUp;
```

Comment:
nice catch!

Reply:
it took a little while to find this one haha, I kept looking in my implementation for the error, and finally found this 😂